flex/tests/rt_mutable_graph/test_graph_loading.cc

```diff
@@ -0,0 +1,60 @@
+/** Copyright 2020 Alibaba Group Holding Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "flex/engines/hqps_db/database/mutable_csr_interface.h"
+
+#include <yaml-cpp/yaml.h>
+
+int main(int argc, char** argv) {
+  if (argc < 4) {
+    std::cout << "Usage: " << argv[0]
+              << "<schema_file> <bulk load file> <data dir>" << std::endl;
+    return 0;
+  }
+
+  setenv("TZ", "Asia/Shanghai", 1);
+  tzset();
+
+  std::string data_path, schema_file, bulk_load_config_path;
+  schema_file = argv[1];
+  bulk_load_config_path = argv[2];
+  data_path = argv[3];
+
+  double t0 = -grape::GetCurrentTime();
+  auto& db = gs::GraphDB::get();
+
+  auto ret = gs::Schema::LoadFromYaml(schema_file, bulk_load_config_path);
+  db.Init(std::get<0>(ret), std::get<1>(ret), std::get<2>(ret),
+          std::get<3>(ret), std::get<4>(ret), data_path, 1);
+
+  t0 += grape::GetCurrentTime();
```

Comment:
Can we remove this unnecessary grape dependencies?

Reply:
[An issue](https://github.com/alibaba/GraphScope/issues/3094) is created and I will work on reducing dependencies and reducing image size.